docs/workforce-management-framework/topics/con-ui-technology.adoc

```diff
@@ -0,0 +1,18 @@
+[id='{context}-ref-ui-technology']
+= The Technology and Structure of the User Interface in a {PROJECT_NAME} Project
+
+The User Interface of {PROJECT_NAME} consists of a series of modules made available through the public NPM registry and built as modules the link:https://angularjs.org/[AngularJS Framework]. 
+
+=== UI Frameworks
+
+The {PROJECT_NAME} modules are intended to be used as part of a regular AngularJS application and used by it as link:https://docs.angularjs.org/guide/module[AngularJS modules] they are kept in a separate, framework-specific repository on link:https://github.com/feedhenry-raincatcher/raincatcher-angularjs/[GitHub]
```

Comment:
Users may not need to know where they are kept- We have separate contributor guide that we going to use for this.

Reply:
Just lets avoid confusion from end users checking this repo.

Reply:
Great, I wasn't sure about this part, any links required?

Going to reword to just mention the framework itself.

Reply:
No links required as people will get applications depending on their support level upstream etc.